web/ui/react-app/package.json

```diff
@@ -84,23 +81,18 @@
     "@types/reactstrap": "^8.7.2",
     "@types/sanitize-html": "^1.20.2",
     "@types/sinon": "^9.0.4",
-    "@typescript-eslint/eslint-plugin": "2.x",
-    "@typescript-eslint/parser": "2.x",
```

Comment:
Just curious, what happened to these two deps, were we just not using them?

Reply:
it's handled by the react-script. So no need to take care of them

Reply:
like that it reduced the number of potentiel conflict between the different dependencies version